feature-libs/quote/components/details/cart/quote-details-cart.component.html

```diff
@@ -1,13 +1,11 @@
 <ng-container *ngIf="{ showCart: showCart$ | async } as state">
-  <div class="cart-toggle" (click)="onToggleShowOrHideCart(!!state.showCart)">
+  <div class="cx-toggle" (click)="onToggleShowOrHideCart(!!state.showCart)">
     <cx-icon
       aria-hidden="false"
       [type]="state.showCart ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
     >
     </cx-icon>
-    <span class="cart-toggle-text">{{
-      'quote.commons.cart' | cxTranslate
-    }}</span>
+    <span class="cx-text">{{ 'quote.commons.cart' | cxTranslate }}</span>
```

Comment:
```suggestion
    <span class="cx-toggle-text">{{ 'quote.commons.cart' | cxTranslate }}</span>
```
i think cx-text is to common, what about cx-toggle-text?